### PR TITLE
Create a TCK results file to upload

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by tevans78** [comment]
@benjamin-confino ignore that long message from LibbyBot, that isn't working properly

File: dev/fattest.simplicity/src/componenttest/topology/utils/TestSuiteResult.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    public String toString() {
+        StringBuilder strBuild = new StringBuilder();
+
+        String pass;
+        if (failures == 0 && errors == 0) {
+            pass = "PASSED";
+        } else {
+            pass = "FAILED";
+        }
+        strBuild.append("Test suite: " + name +  timestamp);
```
Comment (Yas-Franco) [inline dev/fattest.simplicity/src/componenttest/topology/utils/TestSuiteResult.java]:
```suggestion
        strBuild.append("Test suite: " + name +  " "  +  timestamp);
```

**Review by Yas-Franco — CHANGES_REQUESTED** [review]
Other than adding a space between the test suite name and timestamp, it all looks good!

**Comment by benjamin-confino** [comment]
#build

**Review by Yas-Franco — APPROVED** [review]

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Review by tevans78 — APPROVED** [review]

**Pull request merged**